cash-register.js

```diff
@@ -6,42 +6,131 @@
 
 // DONE add new items
 
-// print out receipt - almost done! have to remove items & quantity from bill for 2 voiding functions
+// DONE print out bill
 
+// DONE modify void functions to account for bill
+
+// DONE re-configure bill to have running totals for each type of item, rather than store each instance of the same item separately
+
+// DONE fix void functions in respect to re-configured bill
+
+// make an app out of this!
+/*
+
+v1
+1. create index.html
+2. add a form for inputs: item & quantity
+3. outputs bill
+
+- input for staff discount
+- ability to add new items
+- void last transaction
+- void any item + quantity
+
+*/
 
 var cashRegister = {
 	total: 0,
 	lastItemCost: 0,
+	lastItemName: "",
+	lastItemQuantity: 0,
 	bill: {},
 
 	// add to cash register total, taking in name of item & quantity
 	add: function(itemName, quantity) {
 
+		// keep track of what the last item and quantity was for bill
+		this.lastItemName = itemName;
+		this.lastItemQuantity = quantity;
+
 		// looping through itemList to match name to find price & add to total
 		for (var x in itemList) {
 			if (itemName === itemList[x]["name"]) {
-				console.log(itemList[x]["name"]);
+				console.log(itemList[x]["name"] + ", " + quantity);
```

Comment:
Will do! I'm using console.log to generate 'outputs' from my code, I will definitely have to rewrite to remedy this! Haha